DigitalLearningSolutions.Web.Tests/ControllerHelpers/ControllerContextHelper.cs

```diff
@@ -17,6 +17,7 @@
     public static class ControllerContextHelper
     {
         public const int CentreId = 2;
+        public const int UserId = 1;

```

Comment:
Vlad added this as well in 884 with a different ID, so depending on which gets merged first, one of you will have to make sure tests still work fine.

Reply:
Updated to UserId = 2, to match 884, although I think argument order is different so I still expect merge conflicts